scripts/2_create.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE
+IF NOT EXISTS customers
+(
+    id INTEGER PRIMARY KEY,
```

Comment:
Do these integer IDs come from the source data? Can we use a UUID surrogate key instead?

My concern is that auto-incrementing integer IDs is a SQL only feature and doesn't lend the system to scale in different ways later such as adding multi-primary clustering.

It's not a particular concern for the foreseeable future but I'd like to have DB best practices wherever we can.

Reply:
The non-auto-incrementing IDs are from the source data. We could use UUIDs and have the source ID as a field, but I don't see any value in doing so, as the source is still constrained.

That is a question better answered when we fold this functionality into asset-inventory-api, IMO, but please let me know if you disagree.